### DISCUSSION TITLE
How can I enable c++11 lambda with universal ctags?

### DISCUSSION THREAD

**Discussion by jokoon** [universal-ctags/ctags#3377] [Q&A] [answered]
I remember seeing anonymous function in my tags file once, but I don't remember how I did it.

Any idea?

**Comment by masatake** [comment, marked as answer]
I guess it is the effect of your configuration files under `~/.ctags.d` directory.
Pass '--options=NONE' as the first argument of the ctags command line. 

Even if your trouble is still reproduced with the option, it can be a bug of ctags.
Please, show me a minimum input reproducing the issue. In this case, please remember I don't know well about C++ syntax.
